ts/src/main/scala/ch/epfl/ts/brokers/StandardBroker.scala

```diff
@@ -110,6 +111,11 @@ class StandardBroker extends Component with ActorLogging {
       log.debug("Broker: received order")
       val replyTo = sender
       val uid = o.chargedTraderId()
+      if (!ableToProceed(o)){
+        log.debug("Broker: Unable to proceed MarketBid request before getting first quote")
```

Comment:
Maybe another log level ("info" or "warning"?) would be more appropriate in this case